### ISSUE TITLE
RPI 1644 : Update site to refer to GPL as the only license applicable

### ISSUE THREAD

**Issue by z390org** [z390development/z390#260]
I’m no expert on open source or copyrights.

 

  I did file for copyright on PC390 way back in the 80’s and then ended up selling it to Micro Focus in 1993 including the copyright.  It was all done with help of attorney.

 

The z390 open source has copyright by Automated Software Tools along with open source public license.

Automated Software tools no longer exists.  I think the original 2004 copyright will run out in 2024 if not updated to latest year of open source publication.

 

I just think it would be good to add additional copyright and current year to each source.  We could use our own names as long as the open source public license is also included.

 

Don Higgins

don@higgins.net

www.don-higgins.net

**Comment by adelosa** [comment]
I don't think we should track issues with the websites here.. only the source.. 
The eventual plan is to get the docs generated by the repo become the new website.

There are already existing issues relating to copyright and licensing.
* #236 Replace obsolete copyright
* #113 GPL license

I think this item should be close with reference to probably #236 .

**Comment by abekornelis** [comment]
I'm closing this issue. Issue #236 and Issue #113 should solve the issue. Treating this one as an unneeded duplicate.

**Comment by z390org** [comment]
Anthony,

I'm entirely okay with removing duplicate issues.
Thanks for fixing that. I've closed issue #260.

As for the location where to store the site sources,
that's quite a different question.

Firstly, there's a rather tight connection between site content
and the z390 code content. I've always learned to keep
code and documentation as closely together as possible.

Secondly, there is not always a clear distinction.
E.g. I have a bunch of macros that generate html.
The generated html could be committed to a separate repo,
but it makes more sense (to my mind) to commit
the 'real' source code and treat generated html as
a product of the build process.

I  agree that the source code for the site should be
in a repository. Only, to my mind, these files should
be in a separate directory of the z390 repository.

I do have the sources for the site.
But I'll refrain from uploading them to github
until we have decided where they should go.

Kind regards,
Abe
===



Op 31/05/2021 om 12:01 schreef Anthony Delosa:
